app.js

```diff
@@ -160,15 +160,20 @@ function loadRoutes (app) {
             if(route.constructor !== Object || !route.path || !route.router || !(route.api_version || route.skip_domain)) {
                 throw new TypeError('routes/' + fname + ' does not export the correct object!');
             }
-            // wrap the route handlers with Promise.try() blocks
-            sUtil.wrapRouteHandlers(route.router);
-            // determine the path prefix
-            var prefix = '';
+            // normalise the path and determine the prefix
+            if(route.path[0] !== '/') {
+                route.path = '/' + route.path;
+            }
+            if(route.path[route.path.length - 1] !== '/') {
+                route.path = route.path + '/';
```

Comment:
Because `route.path` is supplied by the user, and it may or may not contain the trailing slash. Alas, express.js adds a front slash to routes, so when gathering the metric's name, we may end up with two slashes. It's faster to check it once here then in `util.js` for every user-supplied route.